openmc/mgxs_library.py

```diff
@@ -1084,10 +1297,9 @@ class MGXSLibrary(object):
     """
 
     def __init__(self, energy_groups):
-        self._xsdatas = []
-        self._energy_groups = energy_groups
+        self.energy_groups = energy_groups
         self._inverse_velocities = None
```

Comment:
You're correct.  I don't know why I got it in my head that this was any different.